MDCNetworking/DynamicCoding.swift

```diff
@@ -131,30 +131,87 @@ public extension KeyedDecodingContainer where Key == DynamicKey {
                 dict[key.stringValue] = value
```

Comment:
AFAIK this should be removed by @bartlomiejn , right?

Reply:
@poksi592  I've asked him, and we still use it.